src/screens/MainScreen/index.js

```diff
@@ -126,6 +134,15 @@ const MainScreen = () => {
           />
         </Animated.View>
       )}
+      {showDeleteConfirmation && (
+        <DeleteConfirmation
+          topicId={selectedTarget.topicId}
+          targetId={selectedTarget.id}
+          targetTitle={selectedTarget.title}
```

Comment:
destructure selectedTarget

Reply:
why don't you send `selectedTarget` instead of separate information? It's better if in the future the DeletConfirmation pop up needs to access other target information.

Reply:
Agree with @ManuViola77 second comment